doc/manuals/admin/build_source.md

```diff
@@ -85,12 +85,12 @@ The Orion Context Broker comes with a suite of unit, valgrind and end-to-end tes
         tar xfvj gmock-1.5.0.tar.bz2
         cd gmock-1.5.0
         ./configure
-        make
         sed -i 's/env python/env python2/' gtest/scripts/fuse_gtest_files.py  # little hack to make installation to work on CentOS 8
+        make
```

Comment:
In the case of the procedure before this update, the following error occurred.
```
(cd lib/.libs && rm -f libgtest_main.la && ln -s ../libgtest_main.la libgtest_main.la)
mkdir -p "./fused-src"
chmod -R u+w "./fused-src"
rm -f "./fused-src/gtest/gtest-all.cc"
rm -f "./fused-src/gtest/gtest.h"
"./scripts/fuse_gtest_files.py" "./fused-src"
env: 'python': No such file or directory
make[1]: *** [Makefile:1230: fused-gtest] Error 127
make[1]: Leaving directory '/gmock-1.5.0/gtest'
make: *** [Makefile:715: all-recursive] Error 1
```

Reply:
You are right. The right order of commands is as you have modified in this PR. Thanks for the fix!

NTC